examples/Calc/CMakeLists.txt

```diff
@@ -20,3 +20,8 @@ if(Boost_UNIT_TEST_FRAMEWORK_FOUND)
     add_executable(BoostCalculatorSteps features/step_definitions/BoostCalculatorSteps)
     target_link_libraries(BoostCalculatorSteps Calc ${CUKE_LIBRARIES} ${Boost_UNIT_TEST_FRAMEWORK_LIBRARY})
 endif()
+
+if(Qt5TEST_FOUND)
+    add_executable(QtTestCalculatorSteps features/step_definitions/QtTestCalculatorSteps)
+    target_link_libraries(QtTestCalculatorSteps Calc ${QT_LIBRARIES} ${CUKE_LIBRARIES})
```

Comment:
I think you can replace `${QT_LIBRARIES}` with `Qt5::Test` here as you only need the test framework, not the GUI stuff.